release.json

```diff
@@ -10,6 +10,17 @@
 		"OMNIBUS_RUBY_BRANCH": "agent-v5",
 		"OMNIBUS_SOFTWARE_BRANCH": "agent-v5"
 	},
+	"5.32.6-rc.1": {
+		"AGENT_VERSION": "5.32.6",
+		"AGENT_BRANCH": "5.32.6-rc.1",
+		"AGENT6_BRANCH": "agent-v5",
+		"INTEGRATIONS_CORE_BRANCH": "agent-v5",
+		"TRACE_AGENT_BRANCH": "agent-v5",
+		"TRACE_AGENT_ADD_BUILD_VARS": "true",
+		"PROCESS_AGENT_BRANCH": "agent-v5",
```

Comment:
has this changed since 5.32.5? Asking because this used to be pinned to the `6.10.0` tag in 5.32.5

Reply:
There are no changes.